tools/ldb_cmd.cc

```diff
@@ -3073,47 +3080,20 @@ void ScanCommand::DoCommand() {
       }
     }
 
-    Slice key_slice = it->key();
-
-    std::string formatted_key;
-    if (is_key_hex_) {
-      formatted_key = "0x" + key_slice.ToString(true /* hex */);
-      key_slice = formatted_key;
-    } else if (ldb_options_.key_formatter) {
```

Comment:
Just curious, this logic that uses `ldb_options_.key_formatter` is removed, is this intended?

Reply:
No it wasn't. Thanks for catching this. Let me add it back.